### DISCUSSION TITLE
High yaw warnings

### DISCUSSION THREAD

**Discussion by tchatte3** [OpenFAST/openfast#878] [Q&A]
Hello openfast development team,

I am having a weird issue with the warning for a current case of floating wind turbine model. 

``Warning: SkewedWakeCorrection encountered a large value of chi (94.039 deg), so the yaw

correction will be limited. This warning will not be repeated though the condition may persist.

See the AD15 chi output channels, and consider turning off the Pitt/Peters skew model (set

SkewMod=1) if this condition persists." ``

My platform degrees of freedom are switched on. The wind direction is 270 degrees (facing the turbine), so there should be no yaw basically. Could this be coming from PtfmYDOF ?

Best Regards,

**Comment by ebranlard** [comment]
Hi @tchatte3 

It is likely that the turbine has some large displacements due to some errors in the model. You can have a look for instance at outputs `PtfmSurge, PtfmSway, PtfmHeave, PtfmRoll, PtfmPitch, PtfmYaw, NacYaw, TTDspFA, TTDspSS, RotSpeed, OoPDefl1, IPDefl1` and `RtSkew`, to see any anomalies. 

If `YawDOF` and `PtfmYDOF` are on, the model needs to have a realistic `PtfmYIner` present, otherwise these degrees of freedom will be ill-defined in ElastoDyn. The Y-inertia, should represent the torsional inertia of the platform/TP (if any) and the one from the tower. You can try to turn off one or both degrees of freedom, and look if that helps. 

Some other common issues with yaw, are when `NacYaw` (in ElastoDyn) and `YawNeut` (in ServoDyn), are in disagreement, ,or, when the yaw spring and damping `YawSpr` and `YawDamp` are too low. 

You can also disable the drive train torsion at first, to eliminate this potential source of error. 

I hope that will help,

Emmanuel

**Comment by tchatte3** [comment]
@ebranlard thank you for the initial direction and guidance.
